Src/GoogleApis.Auth.Tests/OAuth2/Flows/AuthorizationCodeFlowTests.cs

```diff
@@ -76,10 +76,6 @@ public void TestConstructor_DefaultValues()
             Assert.NotNull(flow.HttpClient);
             Assert.NotNull(flow.Scopes);
             Assert.That(flow.TokenServerUrl, Is.EqualTo("https://token.com"));
-
-            Assert.That(flow.HttpClient.MessageHandler.UnsuccessfulResponseHandlers.Count(), Is.EqualTo(1));

```

Comment:
I'm OK with leaving that with a warning, since this test tests for default values.
BackOfHandler is added by default. I prefer to have a test that has warnings in it, than not test it.
Alternative here will be to create a "dummy" service call and check if the BackOffHandler was called.